### ISSUE TITLE
Future of This Project

### ISSUE THREAD

**Issue by edm00se** [edm00se/generator-presto-preso#73]
TL;DR: the functionality and unique features are mostly replaced by either:

- [reveal.js](https://github.com/hakimel/reveal.js)
- [mdx-deck](https://github.com/jxnblk/mdx-deck)

Since I don't want to maintain a large project like this in direct competition with the above projects, I'm attempting to feel out the path of least resistance. Right now, the feature gap I see is primarily the ability to have an auth'd user pushing slide state over websocket to connected listening sessions. This was a somewhat specific use case, so it may not be a deal breaker for for others.

Of the above projects, I would most recommend starting with mdx-deck and seeing if that fits your needs. I love reveal.js, but setup and syntax is slightly more verbose for just throwing some markdown into a file for slides; it's still great and offers an excellent out of the box experience.

**Comment by edm00se** [comment]
Feature set:
- slide deck w/ super powers (old = reveal.js)
- static site generation (ejs powered templates for live site/static build)
- web socket controlled "remote" presentation
- template prefilling GH info (nice, not needed)
- live reload dev server

The [mdx-deck](https://github.com/jxnblk/mdx-deck) has nearly all of the feature set, with the exception of the web socket control. Built on gatsby, it has serve/static build functionality baked in, along with a live reloading dev server. Additionally, the slide content can all be in (an) easy to parse mdx file(s), as opposed to the crunchy ejs presentation behemoth file.

Considering the history of, and active nature, of [this PR](https://github.com/jxnblk/mdx-deck/pull/386), web socket control of slide state appears to be "on the way". Alternatively [this piece](https://dev.to/kevnz/remote-control-an-mdx-deck-presentation-1gmm) outlines how to achieve this functionality with a custom provider.

For a fully customizable approach, one could fork [gatsby-starter-deck](https://github.com/fabe/gatsby-starter-deck) and get a fully customizable gatsby app w/ mdx slides.

Switching to mdx-deck is going to be treated as the preferred update path for the intention of this repo.
